core/src/main/scala/org/apache/spark/network/netty/NettyBlockTransferService.scala

```diff
@@ -30,6 +35,7 @@ import org.apache.spark.network.shuffle.{RetryingBlockFetcher, BlockFetchingList
 import org.apache.spark.network.shuffle.protocol.UploadBlock
 import org.apache.spark.serializer.JavaSerializer
 import org.apache.spark.storage.{BlockId, StorageLevel}
+import org.apache.spark.SparkEnv
```

Comment:
Thanks for pointing it out, I'll change my environment from Eclipse to IntelliJ soon.